### ISSUE TITLE
Non optimal regular expression

### ISSUE THREAD

**Issue by funivan** [kalessil/phpinspectionsea#556]
Do not know if this if false positive or enhancement or not relevant :D 
```php
if (!preg_match('!^\+!', $phone)) {
 $phone = '+' . $phone;
}
```

**Comment by kalessil** [comment]
Let's start with what do you expect here =)

**Comment by funivan** [comment]
`strpos($phone, '+')!==0`

**Comment by kalessil** [comment]
Fixed
